### ISSUE TITLE
ORM failed to write geometry type

### ISSUE THREAD

**Issue by erickhalloween** [cockroachdb/sequelize-cockroachdb#52]
We are migrating database from PG to Cockroach and encountered issue when writing data in geometry type via sequelize ORM.

The ORM model looks like below:

```
import Sequelize from "sequelize-cockroachdb";

const sequelize = new Sequelize("test_db", "test_user", "", {
        host: "test_host_address",
        port: 26257,
        dialect: "postgres",
        logging: true,
    });

const postModel = {
    post_id: {
        type: Sequelize.CHAR(24),
        field: "post_id",
        primaryKey: true,
    },
    removed: {
        type: Sequelize.BOOLEAN,
        defaultValue: false,
    },
    location: {
        type: Sequelize.GEOMETRY("POINT", 4326),
        field: "location",
    }
};
const Post = sequelize.define("post", postModel, {
    schema: "test_schema",
    tableName: "post",
});
```
And when writing to DB, sequelize throws error:
`SequelizeDatabaseError: st_geomfromgeojson(): could not determine data type of placeholder $3`

Turned on the logging, and printed sql and parameters:
```
sql:
INSERT INTO "test_schema"."post" ("post_id","removed","location") VALUES ($1,$2,ST_GeomFromGeoJSON($3)) RETURNING *;

parameters:
[
  '5fff209852fa515454dc1566',
  false,
  { coordinates: [ 116.443987, 39.920843 ], type: 'Point' }
]
```

This worked well with Postgresql. But crashed when using cockroachdb.
Looking for help.

**Comment by awoods187** [comment]
@vy-ton  and @otan it looks like sequelize might not support spatial data types natively. Can you investigate further?

**Comment by otan** [comment]
I don't think we ever tested it, it's been on the TODO list for a bit.

I'd need to dig deeper into sequelize to do this, won't have time to do so for a little bit. Does this work for non-prepared statements?

**Comment by erickhalloween** [comment]
Hi @otan, 

For non-prepared query, it worked. 
For insertion via ORM, it neither worked for JSON nor stringified JSON. So I think the problem could be the parsing part (I guess).

There is no problem when loading data from Cockroach. It parsed to geoJson with no issue. The only problem is to declare it as GEOMETRY in ORM and writes it to DB via ORM.

The current workaround is to declare the ORM attribute as type of TEXT. And whenever write to DB, convert the geoJson object to WKT format in string. 

The interesting part is that the ORM automatically converts the attribute into geoJson object when load via ORM even though it is declared as type TEXT.

**Comment by otan** [comment]
This should be fixed in later point releases of CockroachDB v20.2 and v21.1, but feel free to reopen if not.
